vendor/modules.txt

```diff
@@ -15,6 +15,8 @@ github.com/emicklei/go-restful
 github.com/emicklei/go-restful/log
 # github.com/evanphx/json-patch v4.5.0+incompatible
 github.com/evanphx/json-patch
+# github.com/go-logr/logr v0.1.0
```

Comment:
Is this "#" added automatically?

Reply:
Yes this "#" added automatically in file